src/test/resources/config/application.yml

```diff
@@ -18,6 +18,19 @@
 # SPDX-License-Identifier: Apache-2.0
 #*******************************************************************************/
 
+vas:
+  datasource:
+    host: localhost
+    user: default
+    pass: defaultpassword
+  keycloakClientSecret: defautlSecret
+  keycloakClientId: country-risk-client
+  clientName: country-risk-client
+  authentication-url:
+    token-url: https://centralidp.dev.demo.catena-x.net/auth/realms/CX-Central/protocol/openid-connect/token
```

Comment:
again hard coded config. To me it seems like a dangerous assumption, that the linked systems are always up. If this is for your tests, consider to switch to a scenario, that is fully under your control. Limit external dependencies